docs/getting_started.md

```diff
@@ -101,144 +101,10 @@ Assume that you have already downloaded the checkpoints to the directory `checkp
         --eval mAP
     ```
 
-### Top-down image demo
+### Run demos
```

Comment:
How about keep one simple runnable script here, so that users can try it without leaving this page.

Reply:
OK.